### ISSUE TITLE
EXIF data for HEIC pictures

### ISSUE THREAD

**Issue by blur131** [derf/feh#743]
When viewing HEIC/HEIF pictures in FEH and pressing "e" to display exif data a message "no Exif data in file" is displayed instead. There is exif data. It can be viewed in the terminal using exiftool. Is there any way to increase compatibility in FEH with HEIC files? Since Apple is pushing this as the standard format for all iPhones it is becoming quite common. To have FEH handle both jpg and heic seamlessly would be an improvement, in my opinion.

**Comment by UplandJacob2** [comment]
Yeah, I made a script to convert all my HEIC files to jpeg because of this.
